src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -269,7 +261,16 @@ std::string getEntities
   {
     entities.fill(&parseDataP->qcrs.res);
 
-    TIMED_RENDER(answer = entities.render(ciP, EntitiesResponse));
+    if (entities.oe.code != SccNone)
+    {
+      TIMED_RENDER(answer = entities.oe.toJson());
+      ciP->httpStatusCode = entities.oe.code;
+    }
+    else
+    {
+      TIMED_RENDER(answer = entities.render(ciP, EntitiesResponse));
+      ciP->httpStatusCode = SccOk;
+    }
   }
```

Comment:
Checking entities.oe could be done in entities.render ...
If you move it there, you have this fix for all occasions.
If this is possible, of course.